pyvista/plotting/plotter.py

```diff
@@ -5799,8 +5803,10 @@ def add_arrows(self, cent, direction, mag=1, **kwargs):
 
         >>> import numpy as np
         >>> import pyvista as pv
-        >>> cent = np.random.default_rng().random((10, 3))
-        >>> direction = np.random.default_rng().random((10, 3))
+
+        >>> rng = np.random.default_rng(seed=0)
+        >>> cent = rng.random((10, 3))
+        >>> direction = rng.random((10, 3))
```

Comment:
Let's change seed here.
```suggestion
        >>> rng = np.random.default_rng(seed=1)
        >>> direction = rng.random((10, 3))
```

Reply:
There's no need for that (the `cent` and `direction` parameters will already have different numbers with a single seed value).

Reply:
I didn't know that. Thanks!